tests/api/notify_test.py

```diff
@@ -0,0 +1,139 @@
+from datetime import timedelta
+from unittest.mock import Mock
+
+import pytest
+
+from zino.api.notify import Notification, Zino1NotificationProtocol
+from zino.api.server import ZinoServer
+from zino.state import ZinoState
+from zino.statemodels import EventState, ReachabilityEvent
+from zino.time import now
+
+
+class TestZino1NotificationProtocol:
+    def test_init_should_succeed(self):
+        assert Zino1NotificationProtocol()
+
+    def test_when_unconnected_then_peer_name_should_be_none(self):
```

Comment:
"test_when_not_connected.." is better, surely.

Reply:
Indeed, but the name mirrors the same test for the base server protocol in `legacy_test`, hence the name.  If I change this, both should be renamed.

Reply:
fixed